include/staff/templates/queue-tickets.tmpl.php

```diff
@@ -13,6 +13,23 @@
 )
     $tickets->filter($thisstaff->getTicketsVisibility());
 
+    $tickets->annotate(array(
```

Comment:
This will be a major performance penalty, especially for bigger datasets,  since you're forcing every queue to aggregate data it might not need on every load when we already have a mechanism to inject columns (what to select) on demand. We already can add such counts as columns or as annotation on other columns.